src/main/java/core/basesyntax/figures/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.suppliers.ColorSupplier;
+import java.util.Random;
+
+public class IsoscelesTrapezoid extends Figure {
+    private static final String name = "isosceles trapezoid";
+    private int topSide;
+    private int downSide;
+    private int height;
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final Random random = new Random();
+
+    public IsoscelesTrapezoid() {
+    }
```

Comment:
now you don’t use this variable and this empty constructor too. Let’s remove unused code.